### PR TITLE
Tweaks to the READMEs

### PR DESCRIPTION
**Pull request by nedbat**

### CODE REVIEW

**Comment by nedbat** [comment]
@Lyla-Fischer nothing controversial, I think.

File: README.rst [hunk 1]
```diff
@@ -3,11 +3,11 @@ XBlock SDK |build-status| |coverage-status|
 
 This repository consists of three main components to assist in the creation of new XBlocks:
 
-* a template-based generator for new XBlocks (found in `prototype`)
+* a template-based generator for new XBlocks (found in the ``prototype`` directory)
 
-* sample XBlocks that can be the basis for new XBlock work (found in `sample_xblocks`)
+* sample XBlocks that can be the basis for new XBlock work (found in ``sample_xblocks`` directory)
```
Comment (Lyla-Fischer) [inline README.rst]: "the" for consistency?

File: README.rst [hunk 2]
```diff
@@ -3,11 +3,11 @@ XBlock SDK |build-status| |coverage-status|
...
 
-* Workbench runtime, a simple runtime for viewing and testing XBlocks in a browser (found in `workbench`)
+* Workbench runtime, a simple runtime for viewing and testing XBlocks in a browser (found in ``workbench`` directory)
```
Comment (Lyla-Fischer) [inline README.rst]: "the" for consistency?

File: README.rst [hunk 3]
```diff
@@ -77,7 +77,7 @@ to set the student ID, which defaults to 1::
     http://127.0.0.1:8000/?student=17
 
 Different students will see different student state, for example, while seeing
-the same content.  Student ids are strings, even if they contain only digits
+the same content.  Student IDs are strings, even if they contain only digits
 as the default does.
```
Comment (Lyla-Fischer) [inline README.rst]: This isn't your wording, but it uses implicit grammar, which I find confusing. Maybe something more like:

**Comment by nedbat** [comment]
Updated.

**Comment by Lyla-Fischer** [comment]
lgtm :+1:

**Pull request merged**